wsd/ClientSession.cpp

```diff
@@ -1215,6 +1215,16 @@ bool ClientSession::_handleInput(const char *buffer, int length)
             docBroker->updateLastModifyingActivityTime();
         }
 
+        /*
+            When the UI mode is switched, tile invalidation callback is sent before the mode switch callback.
+            This causes the tiles' ui mode to conflict with the view mode.
+            We set the view mode beforehand here.
+        */
+        if (firstLine == "uno .uno:NormalMultiPaneGUI")
+            _clientSelectedMode = 0;
+        else if (firstLine == "uno .uno:NotesMode")
+            _clientSelectedMode = 2;
```

Comment:
we need enum for that magic number :)
1 was master page mode also IIRC

Reply:
don't we even have enum already?

Reply:
There wasn't an enumerator for the current values. I added.